src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -653,7 +667,7 @@ void enforceMemberNotNull(SyntaxNode? syntaxOpt, LocalState state)
                 }
             }
 
-            void checkMemberStateOnConstructorExit(MethodSymbol constructor, Symbol member, LocalState state, int thisSlot, Location? exitLocation, bool forcePropertyAnalysis)
+            void checkMemberStateOnConstructorExit(MethodSymbol constructor, Symbol member, LocalState state, int thisSlot, Location? exitLocation, ImmutableArray<string> membersWithStateEnforcedByRequiredMembers, bool forcePropertyAnalysis)
```

Comment:
>ImmutableArray<string> [](http://example.com/codeflow?start=148&length=22)

Would it be better to use a HashSet here?

Reply:
I thought about it, but I didn't think it was worth it. I expect this collection to be very small, and while there could certainly be duplicates, my gut feel is that it's not really going to matter.